google/pubsub/v1/pubsub.proto

```diff
@@ -607,7 +607,7 @@ message Subscription {
   // in length, and it must not start with `"goog"`.
   string name = 1 [
       (google.api.required) = true,
-      (google.api.resource_type) = "google.pubsub.v1.Subscription"];
+      (google.api.resource).path = "projects/{project}/subscriptions/{sub}"];
```

Comment:
I think this is needs to be: `projects/*/subscriptions/*`.
I agree it should be `resource` rather than `resource_type`.